src/Admin/Ajax.php

```diff
@@ -149,6 +143,7 @@ public function handle_refresh_keys() {
 			wp_send_json_error(
```

Comment:
can you keep the format for the other modal msg, we'll need to standardize this

Reply:
This is not for athe modal anymore, but for the error TextAlert